src/Resolver/ValueInjection.php

```diff
@@ -120,12 +115,15 @@ public function toValue(ContainerInterface $container)
      * Get the value to inject
      *
      * @deprecated Since 3.1.0
+     *
      * @see toValue()
+     *
+     * @return mixed
      */
     public function getValue()
     {
         trigger_error(
-            __METHOD__ . ' is deprecated, please migrate to ' . __CLASS__ . '::toValue().',
+            __METHOD__ . ' is deprecated, please migrate to ' . self::class . '::toValue().',
```

Comment:
Was that applied by the codestyle? Imho, we should stick with `__CLASS__` constant.

Reply:
phpcbf did this.